realm/realm-library/src/objectServer/java/io/realm/PermissionManager.java

```diff
@@ -151,10 +162,9 @@ public boolean isGlobalRealm() {
     private RealmResults<PermissionOffer> offers;
 
     /**
-     * FIXME Javadoc
      * Creates a PermissionManager for the given user.
      *
-     * Implementation notes: This class is thread safe since all public methods have thread confined checks in them
+     * This class is thread safe since all public methods have thread confined checks in them
```

Comment:
Yeah they downvote me to oblivion on Reddit when I say they're thread-safe because they're thread-confined. I guess people expect "thread-safe == immutable" so I'd just say "this class is thread-confined, and can only be accessed on the thread it was obtained on" or something like that.